### PR TITLE
Fixing tests

### PR DESCRIPTION
**Pull request by kolioOtSofia**
Closes: #Issue

### CODE REVIEW

File: packages/AutoTDD-Core.package/ATDDTestingEntity.class/class/colorDictionary.st [hunk 1]
```diff
@@ -6,5 +6,8 @@ colorDictionary
 			#error -> Color red.
       		#failure -> Color orange.
 			#successful -> Color green.
-			#pending -> Color gray
+			#passed -> Color green.
+			#pending -> Color gray.
+			#errored -> Color red.
+			#failed -> Color orange.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingEntity.class/class/colorDictionary.st]: Remove

File: packages/AutoTDD-Core.package/ATDDTestingEntity.class/class/colorDictionary.st [hunk 2]
```diff
@@ -6,5 +6,8 @@ colorDictionary
 			#error -> Color red.
       		#failure -> Color orange.
 			#successful -> Color green.
-			#pending -> Color gray
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingEntity.class/class/colorDictionary.st]: Reinstate

File: packages/AutoTDD-Dummy-Category.package/ClassGettingTested.class/instance/bla.st [hunk 3]
```diff
@@ -1,4 +1,3 @@
 action
 bla
-	
-	ATDDBrowserTests setTestHasRun
\ No newline at end of file
+ATDDBrowserTests setTestHasRun.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Dummy-Category.package/ClassGettingTested.class/instance/bla.st]: Revert back

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/class/update.st [hunk 4]
```diff
@@ -0,0 +1,6 @@
+constants
```
Comment (SebastianWagner2) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/class/update.st]:
```suggestion
updating
```

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/openEditorForNewTestingEntityOfType..st [hunk 5]
```diff
@@ -1,5 +1,4 @@
 action
 openEditorForNewTestingEntityOfType: aTestingEntitySymbol
 	
-	^ (self class testingEntityEditorDictionary at: aTestingEntitySymbol) openForNewToAddTargetTo:
-		(self testingEntityCollectionDictionary at: aTestingEntitySymbol)
\ No newline at end of file
+	^ (self class testingEntityEditorDictionary at: aTestingEntitySymbol) openForNewToAddTargetTo: (self testingEntityCollectionDictionary at: aTestingEntitySymbol)
```
Comment (SebastianWagner2) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/openEditorForNewTestingEntityOfType..st]: Revert

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/openOverview.st [hunk 6]
```diff
@@ -3,5 +3,5 @@ openOverview
 	
 	self selectedWatchers size + self selectedTravisStatuses size + self selectedGitHubActions size
 		> 0
-		ifTrue: [ ^ ATDDOverview openForWatchers: self selectedWatchers forStatuses: self selectedTravisStatuses forActions: self selectedGitHubActions]
+		ifTrue: [ ^ ATDDOverview openForWatchers: self selectedWatchers asOrderedCollection forStatuses: self selectedTravisStatuses asOrderedCollection forActions: self selectedGitHubActions asOrderedCollection]
```
Comment (SebastianWagner2) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/openOverview.st]: Test if it works without

Comment (SebastianWagner2) [reply packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/openOverview.st]: And change formatting, one testing entity per line

File: packages/AutoTDD-GUI.package/ATDDOverview.class/instance/update.st [hunk 7]
```diff
@@ -0,0 +1,4 @@
+calculation
```
Comment (SebastianWagner2) [inline packages/AutoTDD-GUI.package/ATDDOverview.class/instance/update.st]:
```suggestion
updating
```

File: packages/AutoTDD-GUI.package/ATDDResults.class/instance/colorForStatus..st [hunk 8]
```diff
@@ -1,4 +1,4 @@
 accessing
 colorForStatus: aSymbol
 	
-	^ ATDDTestingEntity colorDictionary at: aSymbol
\ No newline at end of file
+	^ ATDDTestingEntity colorDictionary at: (aSymbol asSymbol)
```
Comment (SebastianWagner2) [inline packages/AutoTDD-GUI.package/ATDDResults.class/instance/colorForStatus..st]: Change signature or fix callers
```suggestion
	^ ATDDTestingEntity colorDictionary at: aSymbol
```

File: packages/AutoTDD-GUI.package/ATDDSoundPlayer.class/class/soundDictionary.st [hunk 9]
```diff
@@ -4,6 +4,6 @@ soundDictionary
 	^ IdentityDictionary newFrom: { 
 		#error -> 'fail' .
 		#failure -> 'fail'.
-
+		#passed -> 'success'.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-GUI.package/ATDDSoundPlayer.class/class/soundDictionary.st]: Remove

Comment (SebastianWagner2) [reply packages/AutoTDD-GUI.package/ATDDSoundPlayer.class/class/soundDictionary.st]: Fix caller

File: packages/AutoTDD-Tests.package/ATDDBrowserTests.class/instance/testOverviewButtonIntegration.st [hunk 10]
```diff
@@ -4,12 +4,12 @@ testOverviewButtonIntegration
 	| overviewWindow watcher travisStatus travisStatusesSelection watcherSelection newBrowser |
 	
 	watcher := ATDDWatcher new.
-	browser addWatcher: watcher.
+	ATDDBrowser Watchers addLast: watcher.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDBrowserTests.class/instance/testOverviewButtonIntegration.st]:
```suggestion
	watcherEditor := browser createWatcher.
	watcherEditor save.
```

File: packages/AutoTDD-Tests.package/ATDDBrowserTests.class/instance/testOverviewButtonIntegration.st [hunk 11]
```diff
@@ -4,12 +4,12 @@ testOverviewButtonIntegration
 	| overviewWindow watcher travisStatus travisStatusesSelection watcherSelection newBrowser |
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDBrowserTests.class/instance/testOverviewButtonIntegration.st]:
```suggestion
	| overviewWindow watcher travisStatus travisStatusesSelection watcherSelection newBrowser watcherEditor |
```

File: packages/AutoTDD-Tests.package/ATDDBrowserTests.class/instance/testOverviewButtonIntegration.st [hunk 12]
```diff
@@ -4,12 +4,12 @@ testOverviewButtonIntegration
...
 	
 	watcher := ATDDWatcher new.
-	browser addWatcher: watcher.
+	ATDDBrowser Watchers addLast: watcher.
 	watcherSelection := OrderedCollection new.
 	watcherSelection add: watcher.
 	browser selectedWatchers: watcherSelection.
 	travisStatus := ATDDTravisStatus new.
-	browser addTravisStatus: travisStatus.
+	ATDDBrowser TravisStatuses addLast: travisStatus.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDBrowserTests.class/instance/testOverviewButtonIntegration.st]: Same thing

File: packages/AutoTDD-Tests.package/ATDDOverviewTests.class/instance/setUp.st [hunk 13]
```diff
@@ -2,9 +2,9 @@ running
 setUp
 	
 	super setUp.
-	watchers := { ATDDWatcher new categoriesToTest: #( 'AutoTDD-Dummy-Category' ) }.
-	travisStatuses := { ATDDTravisStatus new }.
-	gitHubActions := { ATDDGitHubAction new }.
+	watchers := { ATDDWatcher new categoriesToTest: #( 'AutoTDD-Dummy-Category' ) } asOrderedCollection.
+	travisStatuses := { ATDDTravisStatus new } asOrderedCollection.
+	gitHubActions := { ATDDGitHubAction new } asOrderedCollection.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDOverviewTests.class/instance/setUp.st:7]: Check if it works without

File: packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st [hunk 14]
```diff
@@ -9,9 +9,9 @@ testTestsPassingFailingErroring
 	self assert: (resultsWindow historyInfo isKindOf: Text).
 	self assert: ('2 run*, 2 passes, 0 expected failures, 0 failures, 0 errors, 0 unexpected passes'
 		match: resultsWindow historyInfo).
-	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDWatcherResults colorDictionary at: #passed).
+	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDTestingEntity colorDictionary at: #passed).
 	self assert: resultsWindow tests size = 2.
-	self assert: (resultsWindow tests first colorAt: 1) = (ATDDWatcherResults colorDictionary at: #passed).
+	self assert: (resultsWindow tests first colorAt: 1) = (ATDDTestingEntity colorDictionary at: #passed).
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st]:
```suggestion
	self assert: (resultsWindow tests first colorAt: 1) = (ATDDTestingEntity colorDictionary at: #successful).
```

File: packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st [hunk 15]
```diff
@@ -9,9 +9,9 @@ testTestsPassingFailingErroring
 	self assert: (resultsWindow historyInfo isKindOf: Text).
 	self assert: ('2 run*, 2 passes, 0 expected failures, 0 failures, 0 errors, 0 unexpected passes'
 		match: resultsWindow historyInfo).
-	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDWatcherResults colorDictionary at: #passed).
+	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDTestingEntity colorDictionary at: #passed).
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st]:
```suggestion
	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDTestingEntity colorDictionary at: #successful).
```

File: packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st [hunk 16]
```diff
@@ -20,9 +20,9 @@ testTestsPassingFailingErroring
 	self assert: (resultsWindow historyInfo isKindOf: Text).
 	self assert: ('3 run*, 0 passes, 0 expected failures, 3 failures, 0 errors, 0 unexpected passes'
 		match: resultsWindow historyInfo).
-	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDWatcherResults colorDictionary at: #failed).
+	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDTestingEntity colorDictionary at: #failed).
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st]:
```suggestion
	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDTestingEntity colorDictionary at: #failure).
```

File: packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st [hunk 17]
```diff
@@ -20,9 +20,9 @@ testTestsPassingFailingErroring
...
 	self assert: resultsWindow tests size = 3.
-	self assert: ((resultsWindow tests at: 2) colorAt: 1) = (ATDDWatcherResults colorDictionary at: #failed).
+	self assert: ((resultsWindow tests at: 2) colorAt: 1) = (ATDDTestingEntity colorDictionary at: #failed).
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st]:
```suggestion
	self assert: ((resultsWindow tests at: 2) colorAt: 1) = (ATDDTestingEntity colorDictionary at: #failure).
```

File: packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st [hunk 18]
```diff
@@ -31,6 +31,6 @@ testTestsPassingFailingErroring
 	self assert: (resultsWindow historyInfo isKindOf: Text).
 	self assert: ('3 run*, 0 passes, 0 expected failures, 0 failures, 3 errors, 0 unexpected passes'
 		match: resultsWindow historyInfo).
-	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDWatcherResults colorDictionary at: #errored).
+	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDTestingEntity colorDictionary at: #errored).
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st]:
```suggestion
	self assert: (resultsWindow historyInfo colorAt: 1) = (ATDDTestingEntity colorDictionary at: #error).
```

File: packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st [hunk 19]
```diff
@@ -31,6 +31,6 @@ testTestsPassingFailingErroring
...
 	self assert: resultsWindow tests size = 3.
-	self assert: (resultsWindow tests last colorAt: 1) = (ATDDWatcherResults colorDictionary at: #errored)
\ No newline at end of file
+	self assert: (resultsWindow tests last colorAt: 1) = (ATDDTestingEntity colorDictionary at: #errored)
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDWatcherResultsTests.class/instance/testTestsPassingFailingErroring.st]:
```suggestion
	self assert: (resultsWindow tests last colorAt: 1) = (ATDDTestingEntity colorDictionary at: #error)
```

File: packages/AutoTDD-Tests.package/ATDDWatcherTests.class/instance/testAttach.st [hunk 20]
```diff
@@ -7,4 +7,5 @@ testAttach
 	emptyWatcher attach: obj.
 	self assert: (emptyWatcher observers includes: obj).
 	emptyWatcher deattach: obj.
-	self deny: (emptyWatcher observers includes: obj)
\ No newline at end of file
+	self deny: (emptyWatcher observers includes: obj)
+
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Tests.package/ATDDWatcherTests.class/instance/testAttach.st]:
```suggestion
```

**Review by SebastianWagner2 — CHANGES_REQUESTED** [review]

**Review by SebastianWagner2 — APPROVED** [review]

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/class/update.st [hunk 21]
```diff
@@ -0,0 +1,6 @@
+updating
+update
+	
+	self changedWatchers.
+	self changedTravisStatuses.
+	self changedGitHubActions.
```
Comment (benjamin-frost) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/class/update.st]:
```suggestion
	self changedGitHubActions
```

File: packages/AutoTDD-GUI.package/ATDDEditor.class/class/openForNewToAddTargetTo..st [hunk 22]
```diff
@@ -6,7 +6,7 @@ openForNewToAddTargetTo: aCollection
 	self isAbstract ifTrue: [ ^ self error: 'Abstract class. Please do not initialize' ].
 	model := self basicNew
 		target: self emptyTarget;
-		outClosure: [:target | aCollection addLast: target];
+		outClosure: [:target | aCollection addLast: target. ATDDBrowser update];
```
Comment (benjamin-frost) [inline packages/AutoTDD-GUI.package/ATDDEditor.class/class/openForNewToAddTargetTo..st]:
```suggestion
		outClosure: [ :target | aCollection addLast: target. ATDDBrowser update ];
```

File: packages/AutoTDD-Tests.package/ATDDWatcherTests.class/instance/testAttach.st [hunk 23]
```diff
@@ -7,4 +7,4 @@ testAttach
 	emptyWatcher attach: obj.
 	self assert: (emptyWatcher observers includes: obj).
 	emptyWatcher deattach: obj.
-	self deny: (emptyWatcher observers includes: obj)
\ No newline at end of file
+	self deny: (emptyWatcher observers includes: obj)
```
Comment (benjamin-frost) [inline packages/AutoTDD-Tests.package/ATDDWatcherTests.class/instance/testAttach.st]:
```suggestion
	self deny: (emptyWatcher observers includes: obj)
```

**Review by benjamin-frost — CHANGES_REQUESTED** [review]

**Review by SebastianWagner2 — APPROVED** [review]

**Review by benjamin-frost — APPROVED** [review]

**Pull request merged**